share/completions/ansible.fish

```diff
@@ -0,0 +1,106 @@
+# fish 2.2.0 does not include native snippet support. Upgrade to fish >= 2.3.0
+# or append the following code to your ~/.config/fish/config.fish.
+
+# for file in ~/.config/fish/conf.d/*.fish
+#     source $file
+# end
+# ansible
+complete -c ansible -s a -l args -r -f -d "module arguments"
+complete -c ansible -l ask-vault-pass -f -d "ask for vault password"
+complete -c ansible -s B -l background -r -f -d "run asynchronously, failing after X seconds (default=N/A)"
+complete -c ansible -s C -l check -f -d "don't make any changes; instead, try to predict some of the changes that may occur"
+complete -c ansible -s D -l diff -f -d "when changing (small) files and templates, show the differences in those files; works great with --check"
+complete -c ansible -s e -l extra-vars -r -d "set additional variables as key=value or YAML/JSON"
+complete -c ansible -s f -l forks -a "(seq 0 (cat /proc/sys/kernel/pid_max))" -d "specify number of parallel processes to use (default=5)"
```

Comment:
pid_max on my system is 32768. That's overkill. Just do `seq 0 100` or so.

Reply:
Also I have no idea if ansible is available on non-linux systems which might not have /proc/sys/kernel/pid_max.